web-common/src/features/dashboards/stores/metrics-explorer-entity.ts

```diff
@@ -11,75 +11,112 @@ import type { V1MetricsViewFilter } from "@rilldata/web-common/runtime-client";
 
 export interface MetricsExplorerEntity {
   name: string;
-  // selected measure names to be shown
+  /**
+   * selected measure names to be shown
+   */
```

Comment:
Note: the only changes in this file are converting `//` comments to `/** */` comments so that these descriptions flow through to intellisense